scripts/azure-linux_mac-release.yml

```diff
@@ -103,47 +106,47 @@ steps:
     make -j4
     make -C tiledb install
   displayName: 'Build libtiledb'
+  continueOnError: true
 
-- script: |
-      echo $sourceVersion
-      commitHash=${sourceVersion:0:7}
-      echo $commitHash
-      echo "##vso[task.setvariable variable=commitHash]$commitHash" ## Set variable for using in other tasks.
-  env: { sourceVersion: $(Build.SourceVersion) }
-  displayName: Git Hash 7-digit
-
-# Archive files
-# Compress files into .7z, .tar.gz, or .zip
 - task: ArchiveFiles@2
   inputs:
-    rootFolderOrFile: '$(Build.BinariesDirectory)'
+    rootFolderOrFile: '$(Build.Repository.LocalPath)'
     includeRootFolder: false
     archiveType: 'tar' # Options: zip, 7z, tar, wim
     tarCompression: 'gz' # Optional. Options: gz, bz2, xz, none
-    archiveFile: $(Build.ArtifactStagingDirectory)/tiledb-$(ARTIFACT_OS)-$(Build.SourceBranchName)-$(commitHash)-$(ARTIFACT_EXTRAS).tar.gz
+    archiveFile: $(Build.ArtifactStagingDirectory)/tiledb-$(ARTIFACT_OS)-build-dir-$(ARTIFACT_EXTRAS).tar.gz
     replaceExistingArchive: true
     verbose: true # Optional
-  condition: succeeded()
 
+- task: PublishBuildArtifacts@1
+  inputs:
+    pathtoPublish: '$(Build.ArtifactStagingDirectory)/tiledb-$(ARTIFACT_OS)-build-dir-$(ARTIFACT_EXTRAS).tar.gz'
```

Comment:
pathToPublish?

Reply:
Good catch. Its been this way since the beginning but we should follow the spec, update.